cmd/skopeo/utils.go

```diff
@@ -104,6 +113,8 @@ func imageFlags(global *globalOptions, shared *sharedImageOptions, flagPrefix, c
 	fs := pflag.FlagSet{}
 	fs.StringVar(&opts.sharedBlobDir, flagPrefix+"shared-blob-dir", "", "`DIRECTORY` to use to share blobs across OCI repositories")
 	fs.StringVar(&opts.dockerDaemonHost, flagPrefix+"daemon-host", "", "use docker daemon host at `HOST` (docker-daemon: only)")
+	fs.IntVar(&opts.maxRetry, "retry-times", 0, "the number of times to retry")
+	fs.DurationVar(&opts.retryDelay, "retry-delay", 5*time.Second, "time to wait before retry execution")
```

Comment:
I’m sorry, by “something like `imageFlags`” I didn’t mean exactly _inside_ `imageFlags`: `imageOptions` are not great for this, see how this has to ignore `flagPrefix`.

But this can be a new `type retryOptions` with a `func retryFlags` — and the retry implementation can work with a `retryOptions` instead of individual parameters.